fsharp-backend/tests/testfiles/staticassets.tests

```diff
@@ -0,0 +1,547 @@
+  37; 215; 70; 152; 113; 203; 130; 88; 200; 70; 54; 110; 71; 52; 26; 38; 99; 33; 169; 97; 110;
+  71; 73; 99; 44; 20; 83; 46; 43; 27; 33; 11; 26; 37; 229; 178; 224; 156; 239; 123; 31; 106;
+  70; 103; 102; 247; 30; 189; 37; 191; 31; 255; 154; 161; 2; 59; 86; 236; 173; 41; 149; 146;
+  213; 18; 200; 235; 182; 254; 252; 35; 134; 109; 93; 214; 60; 81; 209; 83; 170; 154; 17; 145;
+  107; 109; 253; 249; 54; 28; 5; 56; 218; 190; 124; 223; 164; 56; 73; 94; 96; 216; 164; 170;
+  219; 106; 167; 47; 250; 250; 226; 221; 192; 99; 126; 155; 55; 109; 193; 85; 96; 51; 48; 93;
+  85; 27; 230; 207; 172; 123; 240; 252; 237; 147; 119; 56; 16; 28; 197; 73; 188; 22; 152;
+  194; 48; 171; 118; 23; 127; 24; 106; 25; 33; 46; 197; 181; 56; 18; 92; 25; 170; 25; 173;
+  154; 97; 198; 152; 110; 202; 126; 166; 210; 169; 123; 56; 10; 241; 32; 48; 193; 17; 75;
+  242; 198; 170; 102; 130; 64; 110; 180; 222; 63; 251; 18; 71; 33; 30; 92; 233; 63; 23;
+  3; 151; 248; 11; 33; 30; 68; 81; 52; 214; 24; 115; 76; 85; 51; 34; 210; 222; 211; 211;
+  211; 137; 35; 193; 143; 243; 170; 122; 24; 88; 103; 173; 237; 104; 104; 104; 88; 136;
+  35; 193; 143; 165; 148; 73; 177; 88; 92; 130; 35; 193; 143; 62; 202; 146; 116; 58; 253;
+  16; 71; 33; 126; 236; 23; 145; 15; 214; 218; 140; 136; 92; 239; 236; 236; 124; 138; 163;
+  16; 15; 10; 133; 194; 15; 224; 4; 127; 33; 196; 131; 221; 251; 11; 227; 226; 56; 57; 174;
+  106; 51; 34; 210; 222; 146; 207; 221; 194; 145; 224; 65; 169; 84; 186; 160; 106; 15; 2;
+  89; 107; 237; 205; 157; 205; 221; 117; 56; 18; 252; 88; 76; 153; 73; 18; 91; 135; 35;
+  193; 15; 97; 4; 99; 84; 112; 36; 184; 27; 100; 180; 65; 60; 16; 28; 141; 9; 170; 58; 140;
+  49; 221; 12; 249; 36; 34; 135; 240; 32; 196; 81; 203; 131; 211; 69; 96; 205; 150; 101;
+  187; 107; 194; 32; 252; 210; 218; 119; 190; 136; 7; 66; 5; 154; 14; 220; 173; 74; 205;
+  93; 53; 55; 152; 179; 114; 42; 158; 8; 142; 26; 155; 186; 198; 255; 44; 22; 159; 1; 125;
+  214; 234; 171; 198; 166; 174; 141; 120; 32; 56; 82; 171; 235; 129; 217; 12; 9; 84; 245;
+  40; 30; 8; 142; 140; 240; 141; 209; 190; 225; 129; 224; 202; 200; 109; 49; 230; 22; 160;
+  192; 160; 49; 114; 16; 15; 66; 28; 93; 206; 231; 98; 96; 67; 99; 83; 215; 132; 48; 37;
+  223; 47; 158; 201; 38; 120; 16; 82; 129; 40; 138; 130; 203; 23; 214; 124; 193; 35; 161;
+  50; 91; 235; 235; 235; 247; 68; 81; 84; 141; 39; 66; 5; 10; 133; 66; 171; 170; 222; 49;
+  198; 156; 204; 102; 179; 51; 40; 83; 70; 49; 138; 35; 161; 2; 185; 92; 110; 50; 176; 3;
+  120; 111; 173; 253; 200; 31; 202; 0; 35; 136; 200; 0; 142; 66; 42; 144; 36; 201; 114;
+  99; 76; 190; 183; 183; 247; 51; 35; 164; 210; 225; 158; 56; 182; 223; 85; 147; 89; 34;
+  65; 123; 75; 62; 247; 144; 255; 197; 47; 29; 238; 219; 222; 92; 167; 151; 81; 0; 0; 0;
+  0; 73; 69; 78; 68; 174; 66; 96; 130 ]
+
+
+// baseUrlFor
+[test.baseUrlFor] with ExactCanvasName test-baseurlfor-0
+StaticAssets.baseUrlFor_v0 "abcdefghi" = "https://test-baseurlfor-0.darksa.com/gd57quywk2db6zncs3h2hxgfdtc/abcdefghi"
```

Comment:
where does `gd57quywk2db6zncs3h2hxgfdtc` come from?

Reply:
These are all hashes made from the canvas name

https://github.com/darklang/dark/blob/d88146e9d404ce8aa59e82096f15b044f130a8cf/fsharp-backend/src/LibBackend/StaticAssets.fs#L67